README.md

```diff
@@ -97,28 +97,30 @@ const styles = StyleSheet.create({
-| disableGesture |                       false                       |        `bool`         | Disable swipe gesture                                          |
+| Prop                      |                      Default                      |         Type          | Description                                                          |
+| :------------------------ | :-----------------------------------------------: | :-------------------: | :------------------------------------------------------------------- |
+| data                      |        _not required if children is used_         |        `array`        | Data to use in renderItem                                            |
+| children                  |                         -                         |        `node`         | Children elements                                                    |
+| renderItem                |        _not required if children is used_         |        `func`         | Takes an item from data and renders it into the list                 |
+| onMomentumScrollEnd       |                         -                         |        `func`         | Called after scroll end and the first parameter is the current index |
+| vertical                  |                       false                       |        `bool`         | Show vertical swiper                                                 |
+| index                     |                         0                         |       `number`        | Index to start                                                       |
+| renderAll                 |                       false                       |        `bool`         | Render all the items before display it                               |
+| **Pagination**            |
+| showPagination            |                       false                       |        `bool`         | Show pagination                                                      |
+| paginationDefaultColor    |                       gray                        |       `string`        | Pagination color                                                     |
+| paginationActiveColor     |                       white                       |       `string`        | Pagination color                                                     |
+| paginationStyle           |                        {}                         | `ViewPropTypes.style` | Style object for container                                           |
+| paginationStyleItem       |                        {}                         | `ViewPropTypes.style` | Style object for item (dot)                                          |
+| onPaginationIndexSelected |                         -                         |        `func`         | Called after scroll to item when pagination index selected           |
```

Comment:
Could we change this to `onPaginationSelectedIndex` and the description to `Executed when the user presses the pagination index, similar properties onChangeIndex`